lib/versioning/common.ts

```diff
@@ -13,6 +13,8 @@ export interface NewValueConfig {
   toVersion: string;
 }
 export interface VersioningApi {
+  initModule?(): Promise<void>;
```

Comment:
Maybe I should look at class based versioning over datasources 🤔 This is an okay workaround in the meantime.